.github/workflows/dev-deploy.yml

```diff
@@ -0,0 +1,17 @@
+name: deploy to server
+on:
+  push:
+    branches: [ development ]
+    
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+       - name: execute remote ssh command
+         uses: appleboy/ssh-action@master
+         with:
+           host: ${{ secrets.HOST }}
+           username: ${{ secrets.USERNAME }}
+           key: ${{ secrets.KEY }}
+           port: ${{ secrets.PORT }}
```

Comment:
Change these to DEV_HOST and such, and change them to PROD_HOST for the other one, then ill change it up in the github secrets so that it is all setup correctly

Reply:
fixed

Reply:
also for the other variables lol